### DISCUSSION TITLE
Ubuntu Has Disappeared

### DISCUSSION THREAD

**Discussion by HunterJuneau** [nss-evening-cohort-14/evening-client-side#225] [Help Me!] [answered]
## Your Name

Hunter Juneau

## Have you searched the other help tickets? If not, do that first (Duplicates asks for help will be shamed ceaselessly)

Yes!

## Describe your issue

Ubuntu has disappeared from my VS Code and Windows Terminal. My Terminal in VS Code is not working and the tab for Ubuntu in the Windows Terminal is not there.

## What have you googled? (Paste links to the things you have found from your google search - You must include at least 3 links)

- https://superuser.com/questions/1634922/windows-terminal-no-longer-lists-ubuntu-bash
(That's all I found, but don't really understand what the answer is saying)

## What have you tried? Tell us everything you have tried.

- Restarted computer
- Restarted VS Code
- Checked Windows Terminal

**Comment by ajawashington** [comment, marked as answer]
@HunterJuneau First check your windows terminal settings file and make Ubuntu are still in your environments. If not you will just have to reinstall ubuntu!

**Reply by ajawashington** [reply]
With the new VSCode and Windows update it moves Ubuntu files to new places causing this problem.

**Reply by HunterJuneau** [reply]
I have the separate Ubuntu app but it's not listed in the Windows Terminal.
